assignments/arrays.js

```diff
@@ -63,12 +63,15 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for (let i = 0; i < arr.length; i++) {
```

Comment:
since we already know the position of the car we do not have to loop through. You could do something like this
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
We are looking for car 33 but because the array starts at 0 we need to look at 32.